src/main/java/core/db/DBUpdater.java

```diff
@@ -72,6 +72,12 @@ void updateDB(int DBVersion) {
 	}
 
 	private void updateDBv600(int dbVersion) throws SQLException {
+		// reduce data base file's disk space
```

Comment:
as I have lost my DB I can't check but I imagine those 2 tables are by far the largest. You know you can look into SYSTEM_TABLESTATS to see usage per table ?

Reply:
i found largest ids for those two tables in my database